app/components/primer/flex_item_component.rb

```diff
@@ -3,7 +3,11 @@
 module Primer
   # Use FlexItemComponent to specify the ability of a flex item to alter its
   # dimensions to fill available space
+  #
+  # **Deprecation note:** Use <%= link_to_component(Primer::BoxComponent) %> instead.
```

Comment:
What do you think about providing an example before and after for the suggested migration?

Reply:
Mind if I add that in a follow-up PR? I'm planning on doing something similar to the `@accessibility` custom tag so we can have a section on deprecation